src/index.scss

```diff
@@ -202,8 +202,7 @@
   --bg-surface-extra-low-transparent: hsla(0, 0%, 91%, 0);
 }
 
-.dark-theme,
-.butter-theme {
+@mixin dark-mode() {
   /* background color | --bg-[background type]: value */
```

Comment:
instead of creating a  mixin we can keep here .dark-theme only.

And @ extend .dark-theme in .butter-theme and .auto-theme.
Benefit of doing this is that scss compiler will not copy all code of .dark-theme to auto and butter theme and instead it will group those classes together.